src/migrations/20230828103020-program-personnel-cleanup.js

```diff
@@ -10,35 +10,29 @@ module.exports = {
       await prepMigration(queryInterface, transaction, sessionSig);
 
       await queryInterface.sequelize.query(`
-         /* 1. Backup all current program personnel records in a table */
-         SELECT * INTO "ProgramPersonnel_backup_20230828" FROM "ProgramPersonnel"
-
-
-         /* 2. Create a table of everything we want to keep */
-         DROP TABLE IF EXISTS "ProgramPersonnelToKeep";
-         CREATE TEMP TABLE "ProgramPersonnelToKeep" AS (
-            SELECT
-                 "firstName",
-                 "lastName",
-                 "role",
-                 "grantId",
-                 "programId",
-                 "active",
-                 count(id),
-                 min("id") AS "idToKeep"
-                 FROM "ProgramPersonnel"
-                 GROUP BY
-                 "firstName",
-                 "lastName",
-                 "role",
-                 "grantId",
-                 "programId",
-                 "active"
-                 order by "active" asc
+         /* 1. Remove records we don't want */
+         WITH initial_entries AS (
+         SELECT
+           "firstName",
+           "lastName",
+           "role",
+           "grantId",
+           "programId",
+           active,
+           MIN(id) AS first_insert
+         FROM "ProgramPersonnel"
+         GROUP BY 1,2,3,4,5,6
          )
+         DELETE FROM "ProgramPersonnel"
+         WHERE id NOT IN (SELECT first_insert FROM initial_entries);
+         
+         /* 2. Make a copy of the audit log before cleaning it up */
+         SET session_replication_role = replica;
+         CREATE TABLE "ZZarchiveZALProgramPersonnel20230828103020" AS SELECT * FROM "ZALProgramPersonnel";
```

Comment:
This is a real table and have an audit log generated for it. I don't think that is what is intended.